crates/common/src/header.rs

```diff
@@ -24,14 +24,26 @@ pub struct BlockHeader {
     pub l1_da_mode: L1DataAvailabilityMode,
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Default, Dummy, serde::Serialize, serde::Deserialize)]
+#[derive(
+    Debug, Copy, Clone, PartialEq, Eq, Default, Dummy, serde::Serialize, serde::Deserialize,
+)]
 #[serde(rename_all = "UPPERCASE")]
 pub enum L1DataAvailabilityMode {
     #[default]
     Calldata,
     Blob,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Default, Dummy)]
+pub struct PendingBlockHeader {
+    pub parent_hash: BlockHash,
+    pub timestamp: BlockTimestamp,
+    pub eth_l1_gas_price: GasPrice,
+    pub strk_l1_gas_price: GasPrice,
```

Comment:
Data gas prices and `l1_da_mode` seem to be missing here.

Reply:
Indeed; I'd added this before the v0.13.1 changes hit main 👍 I'll probably be removing this type entirely for this PR.